controllers/storagecluster/reconcile.go

```diff
@@ -396,14 +396,16 @@ func (r *StorageClusterReconciler) reconcilePhases(
 	// enable metrics exporter at the end of reconcile
 	// this allows storagecluster to be instantiated before
 	// scraping metrics
-	if err := r.enableMetricsExporter(instance); err != nil {
-		r.Log.Error(err, "failed to reconcile metrics exporter")
-		return reconcile.Result{}, err
-	}
+	if instance.Spec.MetricsExporter != nil && ReconcileStrategy(instance.Spec.MetricsExporter.ReconcileStrategy) != ReconcileStrategyIgnore {
```

Comment:
I believe this will change the current default behaviour which is to enable metrics exporter.
After this change when `instance.Spec.MetricsExporter == nil` the first half of the if statement will fail and the if block will be skipped which is the opposite of the desired behaviour

Reply:
....ahhhh, yup! Fixed. I hope.